next-docs/pages/world/introduction.mdx

```diff
@@ -1 +1,24 @@
 # Introduction
+
+[`Store`](/store/introduction) provides low-level primitives for storage.
+`World` provides the primitives for logic and access control on top of that storage.
+
+Calls are sent to the `World` as a central entry point.
+The `World` performs access control checks and if the request is authorized, sends it to the appropriate [`System`](/world/systems).
+In most cases access from the `System` to [the tables that store the data](/world/tables) also goes through the `World`, making it impossible to bypass access control.
+The `System` contracts themselves are stateless, all the data is managed by the `World`.
+
+**Note:** A `System` in the root namespace _could_ bypass the access control.
+However, only the owner of the root namespace (by default, the account that deployed the `World`) can deploy into the root namespace.
+If you want to prove that only an authorized `System` will ever be able to change table data, you can revoke all access to the root namespace and change the owner to `address(0)`.
```

Comment:
```suggestion
To prevent systems from being registered in the root namespace, the root namespace owner can [transfer ownership](/world/namespaces-access-control) to `address(0)`.
```